src/components/Header/Header.style.js

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+
+const HeaderWrapper = styled.header`
+  display: flex;
+  height: 65px;
+  width: 100%;
+`;
+
+const LogoContainer = styled.div`
+  display: flex;
+  align-items: center;
+  width: auto;
+  margin-left: ${(props) => props.theme.margin};
+`;
+
+const LinksContainer = styled.nav`
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  width: 100%;
+  margin-right: ${(props) => props.theme.margin};
+`;
+
+const Link = styled.a`
```

Comment:
You're using react-router, right? Then you should use the Link component from that package instead of `styled.a`